src/components/common/form/error-summary/ErrorSummary.jsx

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import { HashLink } from 'react-router-hash-link';
 
-const ErrorSummary = ({ errors }) => {
+const ErrorSummary = ({ errors, keys }) => {
```

Comment:
I think this would be desired, but requires complex logic, we should tackle this in the future

Reply:
I think a potential way to do it would be to have that sort method sort the actual errors from the error object into an ordered array of errors. E.g. `[{error1, msg1},   {error2, msg2},    {error3 msg3}]` something along those lines.
Then feed that into the error summary object and just update how we extract it. Just an idea.

Perhaps we can create another ticket to cover tackling that and try and implment it early on before more components start getting built. @holly-blues Thoughts?

Reply:
@daleelR like that idea 
@LaurieBennett agree we have more work to do on error handling, so can get our heads together and tackle this one at that point